src/components/Composer/index.ios.js

```diff
@@ -97,7 +97,9 @@ function Composer({shouldClear, onClear, isDisabled, maxLines, forwardedRef, isC
      * @return {Number}
      */
     const maxNumberOfLines = useMemo(() => {
-        if (isComposerFullSize) return undefined;
+        if (isComposerFullSize) {
+            return undefined;
```

Comment:
NAB since this was already here, but why `return undefined;` instead of `return;`? should we update it?

Reply:
Done, also merged the newest main